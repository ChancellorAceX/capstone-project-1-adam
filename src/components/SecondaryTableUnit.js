import React from 'react';

export default function SecondaryTableUnit(props) {
  return (
    <tr>
      <th>{props.unit[0]}</th>
      <th>{props.unit[1]}</th>
    </tr>
  )
}