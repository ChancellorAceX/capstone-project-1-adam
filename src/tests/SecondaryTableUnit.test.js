import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import SecondaryTableUnit from '../components/SecondaryTableUnit';

describe('SecondaryTableUnit component', () => {
const props = {
  "unit": [
    "Resistances",
    "radiant; bludgeoning, piercing, and slashing from nonmagical weapons"
  ]
};

  it('renders the full SecondaryTableUnit component', () => {
    const wrapper = shallow(<SecondaryTableUnit {...props} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});