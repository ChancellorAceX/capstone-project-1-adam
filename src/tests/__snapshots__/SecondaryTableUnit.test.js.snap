// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SecondaryTableUnit component renders the full SecondaryTableUnit component 1`] = `
<tr>
  <th>
    Resistances
  </th>
  <th>
    radiant; bludgeoning, piercing, and slashing from nonmagical weapons
  </th>
</tr>
`;
